.scripts/regression-compare.yaml

```
@@ -79,7 +79,7 @@ languages:
       - --package-name:test-package
       - --title:TestClient
       - --use:@autorest/typescript@6.0.0-dev.20200826.1
```

This one can also be updated

```suggestion
      - --use:@autorest/typescript@6.0.0-dev.20201208.1
```